### PR TITLE
Tell the observer if the download is coming from a private tab

### PR DESCRIPTION
**Pull request by ilpianista**
Follow up of [sailfish-browser!1057](https://github.com/sailfishos/sailfish-browser/pull/1057) to propagate the private mode information.

### CODE REVIEW

File: jscomps/EmbedliteDownloadManager.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const Cu = Components.utils;
 const Cr = Components.results;
 
 const { XPCOMUtils } = ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
+const { PrivateBrowsingUtils } = ChromeUtils.import("resource://gre/modules/PrivateBrowsingUtils.jsm");
```
Comment (rainemak) [inline jscomps/EmbedliteDownloadManager.js:18]: Not needed as download object itself carries this information. See https://github.com/sailfishos-mirror/gecko-dev/blob/600df6aa643726626d2eab36bc0f6e34bbe5ac2a/toolkit/components/downloads/Downloads.jsm#L80

**Review by rainemak — CHANGES_REQUESTED** [review]
Not quite there yet but close. I think idea to delivere private status to the user interface is good and should be implemented. Provided some guidelines and instructions how this should be improved.

**Comment by rainemak** [comment]
Any comments?

**Comment by ilpianista** [comment]
> Any comments?

I would like to implement your changes, but had no time to work on it. Maybe next week :crossed_fingers:

File: jscomps/EmbedliteDownloadManager.js [hunk 2]
```diff
@@ -89,6 +91,7 @@ let DownloadView = {
                                    JSON.stringify({
                                        msg: "dl-fail",
                                        id: download.id,
+                                       privateMode: PrivateBrowsingUtils.isContentWindowPrivate(Services.ww.activeWindow) || false,
```
[the remark this answers is not in the document]
Comment (ilpianista) [reply jscomps/EmbedliteDownloadManager.js]: I'm not sure I understood this, please have a look to my changes. Thanks!

**Comment by rainemak** [comment]
> I would like to implement your changes, but had no time to work on it. Maybe next week 🤞

Thanks, it's almost there. Easier to explain in form of PR this https://github.com/sailfishos/embedlite-components/pull/98 so I created one. Point is that we need to pass privateMode update from each download message that we send from the embedlite-components to browser app.

**Pull request closed without merging**